.github/setup-unit.sh

```diff
@@ -1 +1,66 @@
-echo "kestra.git.pat: $GH_PERSONAL_TOKEN" > src/test/resources/application-test.yml
\ No newline at end of file
+#!/bin/bash
+set -e
+
+mkdir -p certs
```

Comment:
When you'll do the release (backport?) ensure this folder will be deleted. I added a clean-up recently so it should be ok but let's be aware of it :)